tools/cravatool/cravat_submit.xml

```diff
@@ -1,33 +1,117 @@
+          <option value="Breast">Breast</option>
+          <option value="Cervix">Cervix</option>
+          <option value="Colon">Colon</option>
+          <option value="GID">GID</option>
+          <option value="Head-and-Neck">Head-and-Neck</option>
+          <option value="Kidney-Chromophobe">Kidney-Chromophobe</option>
+          <option value="Kidney-Clear-Cell">Kidney-Clear-Cell</option>
+          <option value="Kidney-Papillary-Cell">Kidney-Papillary-Cell</option>
+          <option value="Liver-Nonviral">Liver-Nonviral</option>
+          <option value="Liver-Viral">Liver-Viral</option>
+          <option value="Lung-Adenocarcinoma">Lung-Adenocarcinoma</option>
+          <option value="Lung-Squamous-Cell">Lung-Squamous-Cell</option>
+          <option value="Other">Other</option>
+          <option value="Ovary">Ovary</option>
+          <option value="Pancreas">Pancreas</option>
+          <option value="Prostate-Adenocarcinoma">Prostate-Adenocarcinoma</option>
+          <option value="Rectum">Rectum</option>
+          <option value="Skin">Skin</option>
+          <option value="Stomach">Stomach</option>
+          <option value="Thyroid">Thyroid</option>
+          <option value="Uterus">Uterus</option>
+        </param>
+      </when>
+    </conditional>
+    <param format="tabular" name="GRCh" type="select" label="Genome Reference Consortium Human Build (GRCh)" help="The default human reference genome used for annotation is GRCh38, released on December 24th, 2013 from the Genome Reference Consortium.">
       <option value="GRCh38">GRCh38/hg38</option>
       <option value="GRCh37">GRCh37/hg19</option>
     </param>
-    <param format="tabular" name="psm" type="data" optional="true" label="ProBED File(Optional)"> </param>
-    
-    
   </inputs>
   
   <outputs>
-      <collection name="collection" type="list" label="CRAVAT Results: ${on_string} using ${dropdown}">
-        <data format="cravat" label="CRAVAT: Gene Level Annotation Report" name="Gene" />
-        <data format="cravat" label="CRAVAT: Variant Report" name="Variant" />
-        <data format="cravat" label="CRAVAT: Non-coding Variant Report" name="Noncoding" />
-        <data format="cravat" label="CRAVAT: Errors" name="Error" />
+      <collection name="collection" type="list" label="CRAVAT Results on ${on_string}">
+        <data format="tabular" label="CRAVAT: Gene Level Annotation Report on ${on_string}" name="Gene" />
+        <data format="tabular" label="CRAVAT: Variant Report on ${on_string}" name="Variant" />
+        <data format="tabular" label="CRAVAT: Non-coding Variant Report on ${on_string}" name="Noncoding" />
+        <data format="tabular" label="CRAVAT: Errors on ${on_string}" name="Error" />
       </collection>
+      <data format="vcf" label="Intersected VCF on ${on_string}" name="vcf_output">
+        <filter>proteo['proteoInput'] and proteo['output_vcf']</filter>
+      </data>
   </outputs>
 
   <tests>
```

Comment:
You really need to have more tests here, at least one for each conditional. This will also test your python code.